### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by elpotatoslayer**
- create a react app.
- setup linters.
- published to netlify.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Live Link
+Click here for [live link](https://romantic-williams-bee2ec.netlify.app/)
+
+## Available Scripts
+
+In the project directory, you can run:
+
```
Comment (addod19) [inline README.md:11]: - [ ] It would be better if we can add instructions about how to clone the app locally, so non-developers can also work with the repo. It would also be great to include your social media handles, etc. Let us kindly use the [Microverse template](https://github.com/microverseinc/readme-template) as inspiration

File: package.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "calaculator-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "big.js": "^6.1.1",
+    "react": "^17.0.2",
```
Comment (addod19) [inline package.json:10]: - [ ] It would be better if we add the `prop-types` package to the project. This will help us to type check our props

File: public/robots.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (addod19) [inline public/robots.txt:3]: - [ ] It would be better if we delete all the files that are not related to the project. So that the codebase looks cleaner

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @Ralph-1 , 

Good job so far! Student follows Github flow :muscle: Deployed app :rocket: 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

__Happy coding__ :man_technologist: 
[Daniel Larbi Addo]()

File: README.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
+# Getting Started with Create React App
```
Comment (rindrajosia) [inline README.md:1]: - [ ] [OPTIONAL] Please update the title of this readme

File: README.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (rindrajosia) [inline README.md:3]: - [ ]  [OPTIONAL] Please provide a little description about this project

File: README.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+### Tests 
+- Run `npm run test` to run all the tests, or
+- Run `npm run test:watch` to watch the tests
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```
Comment (rindrajosia) [inline README.md:31]: - [ ]  [OPTIONAL] Please add information about yourself like links to your GitHub and LinkedIn

**Review by rindrajosia — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Please consider to:

- [ ] Add information about the Authors, update the project title and add a little description about the project in order to make your readme more descriptive.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
